### PR TITLE
ci: add recent Node.js versions to CI

### PR DESCRIPTION
**Pull request by addaleax**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -16,6 +16,8 @@ node_js:
   - "6"
```
Comment (gengjiawen) [inline .travis.yml]: I think we can remove 6 and 8, since they are EOL.

Comment (addaleax) [reply .travis.yml]: Yeah, but that’s a semver-major change and I’d like to avoid that at this point.

Comment (gengjiawen) [reply .travis.yml]: Since Node.js 13.6.0 comes out, maybe add 13 too.

Comment (addaleax) [reply .travis.yml]: Unfortunately, tests don’t pass on 13 yet

Comment (gengjiawen) [reply .travis.yml]: Why the stack pointed to Node.js source code on Node.js 13 ?
```console
  1) Callback
       "after each" hook for "should create a C function pointer from a JS function":
     TypeError: Cannot read property 'call' of undefined
      at processImmediate (internal/timers.js:456:21)
```

Comment (gengjiawen) [reply .travis.yml]: Ok, find the cause 
https://github.com/node-ffi-napi/node-ffi-napi/blob/18a2a9f0e93e83cc3ee710b920237ebe14da04d1/test/variadic.js#L9

Comment (gengjiawen) [reply .travis.yml]: Okay, this is macOS error. On linux it's another error.

**Comment by pimterry** [comment]
Just a quick note on this (I've been playing with ffi-napi for my own use case, and I wanted to include this PR): the issue that's breaking the windows builds here is https://github.com/istanbuljs/nyc/issues/1205, and it's fixed in nyc v15.

Upgrading to that makes the build & tests all pass immediately for me with Windows x64 & Node 12.13.1.

The one problem there is that nyc 15 drops support for node 6, so you probably need to either need to drop node 6 from testing or add a line in the appveyor builds to switch the nyc version just for that build, as a quick hacky fix. Hope that's useful!

**Pull request closed without merging**